### PR TITLE
Update README and customize profile

### PR DESCRIPTION
**Pull request by aychin**
Update README as part of week 0 setup

### CODE REVIEW

**Comment by aychin** [comment]
Meant to have readme in separate pull request but oh well

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,18 +2,24 @@
...
+      <p>I live in Fremont, CA. I'm originally from NYC and NJ. I like thinking about how things work.</p>
+      <p>I've worked on: TI-83 prime factorizer (high school), course scheduler (undergrad),
+          prime brokerage systems (GS), portfolio management (AQR), credit risk systems (BofA),
+          Google My Business and Local Shopping (Google)</p>
+      <p>My pictures tend to be of nature and food.</p>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <p>Hidden talent: ask Kevin Workman</p>
```
Comment (emilyzhang242) [inline portfolio/src/main/webapp/index.html:22]: Who is Kevin Workman?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,18 +2,24 @@
...
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link rel="icon" type="image/png" href="/images/profile-small.jpg">
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Alan's Webpage</h1>
+      <p>I live in Fremont, CA. I'm originally from NYC and NJ. I like thinking about how things work.</p>
```
Comment (emilyzhang242) [inline portfolio/src/main/webapp/index.html:13]: Fremont is cool!

**Review by emilyzhang242 — APPROVED** [review]
LGTM

**Pull request merged**